reference/release-notes/1.22.0.md

```diff
@@ -0,0 +1,88 @@
+
+* VirGL is the default renderer for NVIDIA GPUs with driver version 545 and later. For driver versions earlier than 545 or old NVIDIA GPU architecture such as Turing, Ampere or Ada, EmuGL is used as the default renderer. See [Rendering architecture](https://anbox-cloud.io/docs/explanation/rendering-architecture).<!--AC-2459, AC-2270 and AC-2348-->
+* Experimental support for [pointer lock](https://developer.mozilla.org/en-US/docs/Web/API/Pointer_Lock_API) is available with this release. This helps in locking the pointer to a particular browser element, for example, your game window, and limits movement outside the game until the lock is disabled. The pointer lock property must be explicitly enabled on the stream object via the [streaming SDK](https://github.com/canonical/anbox-streaming-sdk).<!--AC-2379-->
+* All components of Anbox Cloud upgrade to NVIDIA 550 driver series by default on Ubuntu 22.04 images. For Ubuntu 20.04 images, 535 driver series will be used. <!--AC-2309-->
+
+### Dashboard improvements
+
+The following improvements are available in the Anbox Cloud dashboard for this release:
+
+* Page level guided tours are now available for help in navigating through the dashboard.
+* Support for using custom authentication server instead of Ubuntu SSO.
+* Custom instance name field in *Create instance* form.
+* User experience enhancements to the AAOS control panel:
+  - Extending VHAL support to allow setting the geographic location and to be able to provide and revoke consent for enabling APIs in the browser when using maps.
+* Enhancements to forms to support AOSP vs AAOS applications.
+
+### Other
+
+* Support for building custom images of Anbox Cloud based on AAOS. See [Custom images](tbd).<!--AC-2458-->
+* Android security updates for May 2024 (see [Android Security Bulletin - May 2024](https://source.android.com/docs/security/bulletin/2024-05-01) for more information).<!--AC-2437-->
+* The Android WebView has been updated to [124.0.6367.82](https://chromereleases.googleblog.com/2024/04/chrome-for-android-update_24.html).
+
+## Deprecations
+
+See [Deprecation notices](tbd) for a list of deprecations in this release.
+
+## Removed functionality
+
+* Documentation on how to enable VirGL manually is removed because with this release, VirGL is the default renderer for NVIDIA GPUs.
+
+## Known issues
+
+The following known issues exist in the 1.22.0 release and are planned to be fixed in future releases:
+
+* Video playback does not work on 64-bit only systems where the NVIDIA GPU is installed.
+* For existing charmed deployments, creating instances with streaming enabled using the `enable-streaming` option requires manually adding the new relation between the AMS and the stream agent charms. See [How to upgrade Anbox Cloud](https://anbox-cloud.io/docs/howto/update/upgrade-anbox).
```

Comment:
@keirthana we found one more issue today which we need to record here. Can you add the following:

* Due to a bug in LXD (see [#13420](https://github.com/canonical/lxd/issues/13420) a manual workaround need to applied to every LXD node deployed via the [Anbox Cloud LXD charm](https://charmhub.io/ams-lxd) as otherwise instances launched via AMS will fail with an error similar to the following one `Error: Failed creating instance from image: Failed to run: zfs set volsize=3000000512 ams0/containers/ams-cp1pjd7lra53mbq42mug: exit status 255 (cannot set property for 'ams0/containers/ams-cp1pjd7lra53mbq42mug': 'volsize' must be a multiple of volume block size (16K))`. Once a new unit of the charm has been deployed run `juju ssh lxd/n -- lxc storage set ams0 volume.zfs.block_mode=false`.

Reply:
@jat-canonical @adglkh @ajanon ^^

Reply:
Done